Makefile

```diff
@@ -20,6 +20,9 @@ system:
 	for extension in $(SystemJS); do \
 		echo $$extension"(System);" >> dist/$@.src.js; \
 	done
+	for extension in $(SystemJS); do \
+		echo "System._extensions.push("$$extension");" >> dist/$@.src.js; \
+	done
```

Comment:
We can rename the internal `register` function in the register extension to `registerModule` or something to make this work.